### PR TITLE
Preparations for new release

### PR DESCRIPTION
**Pull request by vytis**
### Short description 📝

A new release is scheduled for this Friday and this PR adds the initial version of the blog post about it

### CODE REVIEW

File: website/markdown/posts/2020-01-31-version-1.2.0/post.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+We are slowly learning how to do smaller [bi-weekly releases](https://tuist.io/docs/contribution/release/#cadence). Everything went (almost) according to plan and Tuist 1.2.0 is here for you to enjoy!
+
+## New contributors
+
+Firstly, warm welcome to another first-time Tuist contributor [Maciej Piotrowski](https://github.com/paciej00) 👏👏👏. He improved the API introduced in the previous release for excluding files from a target. It makes exclude patterns more flexible which can be very useful when migrating an existing project to Tuist!
+
+## TBD
+
+TBD, maybe a few sentences about fixture generator?
```
Comment (pepibumur) [inline website/markdown/posts/2020-01-31-version-1.2.0/post.mdx]: I think it's a good idea. We can mention that we disregarded performance for a long time and now we'd like to tackle it. For that we developed a fixture generator to generate projects that we can use from performance tests.

**Comment by vytis** [comment]
@pepibumur I've updated the post with the new things that happened today. You guys are hard to catch up with! 😅

**Comment by vytis** [comment]
Maybe someone could take a final look before merging? @tuist/core

**Comment by kwridan** [comment]
👍 Looks great thanks for putting this together @vytis

**Pull request merged**